api/test/e2e/global_rule_test.go

```diff
@@ -282,3 +287,109 @@ func TestGlobalRule(t *testing.T) {
 		testCaseCheck(tc, t)
 	}
 }
+
+func TestGlobalRule_with_createtime_updatetime(t *testing.T) {
+	tests := []HttpTestCase{
+		{
+			Desc:   "create global rule",
+			Object: ManagerApiExpect(t),
+			Path:   "/apisix/admin/global_rules/1",
+			Method: http.MethodPut,
+			Body: `{
+                                "plugins": {
+		                        "response-rewrite": {
+		                            "headers": {
+		                                "X-VERSION":"1.0"
+		                            }
+		                        },
+					"uri-blocker": {
+						"block_rules": ["select.+(from|limit)", "(?:(union(.*?)select))"]
+					}
+                                }
+                        }`,
+			Headers:      map[string]string{"Authorization": token},
+			ExpectStatus: http.StatusOK,
+			Sleep:        sleepTime,
+		},
+	}
+
+	for _, tc := range tests {
+		testCaseCheck(tc, t)
+	}
+
+	basepath := "http://127.0.0.1:9000/apisix/admin/global_rules/1"
+	time.Sleep(time.Duration(1) * time.Second)
+
+	// get the global_rule, save createtime and updatetime
+	request, _ := http.NewRequest("GET", basepath, nil)
+	request.Header.Add("Authorization", token)
+	resp, err := http.DefaultClient.Do(request)
+	assert.Nil(t, err)
+	defer resp.Body.Close()
+
+	respBody, _ := ioutil.ReadAll(resp.Body)
+	createtime := gjson.Get(string(respBody), "data.create_time")
+	updatetime := gjson.Get(string(respBody), "data.update_time")
```

Comment:
It would be better if you can compare the current timestamp, refer:

https://github.com/apache/apisix-dashboard/blob/master/api/test/e2e/route_with_management_fileds_test.go#L88-L91

Reply:
@nic-chen Done and thanks for your suggestion.